cypress/page/login.page.ts

```diff
@@ -1,17 +1,17 @@
 class LoginPage {
-    private userEmail: string;
-    private userPassword: string;
+    private Email: string;
+    private Password: string;
```

Comment:
variable names are in camelCase, which means that first letter should be in lower case 😄

Reply:
Oh My God, I am so careless!!!!!
When I corrected your last commentary I did not correct this part.

I am so sorry!